test/integration/ItemPrice.test.ts

```diff
@@ -165,4 +224,71 @@ describe('ItemPrice', () => {
         );
     });
 
+    test('Should create a new item price missing shipping price', async () => {
+        const shippingPrice = testData['shippingPrice'];
+        delete testData['shippingPrice'];
+
+        const itemPriceModel: ItemPrice = await itemPriceService.create(testData);
+        createdId = itemPriceModel.Id;
+
+        testData['shippingPrice'] = shippingPrice;
+
```

Comment:
instead of testData['shippingPrice'] = shippingPrice;
prefer: testData.shippingPrice = shippingPrice;

because, we should try to avoid using objects with 'any' type, they are confusing and you cannot quickly tell what is the information they should or could contain, or what they are for, by any other way than by their naming or looking around the code trying to guess where someone added some magical field which then causes some magical behaviour 6 months from now. :)

if you cast testData as ItemPriceCreateRequest as you have, you should be able to do testData.shippingPrice. If ItemPriceCreateRequest contains shippingPrice (as it should), there should be no errors. And if your ide shows errors and you know there should be shippingPrice in the object, then you know the ItemPriceCreateRequest type is not correctly defined.